babel.config.js

```diff
@@ -3,7 +3,10 @@ module.exports = function (api) {
     api.cache(true);
 
     const presets = [ ["react-app", { "absoluteRuntime": false }] ];
-    const plugins = [ ["babel-plugin-styled-components"] ];
+    const plugins = [
+        ["babel-plugin-styled-components"],
+        ["import", { "libraryName": "lodash", "libraryDirectory": "", "camel2DashComponentName": false }]
```

Comment:
Just so I'm clear as to what this is doing, it's overriding a lodash setting directly, so it won't collide with an ESlint setting in react-payments, or ...?

Reply:
Hey @AndyEPhipps. I installed a babel plugin [babel-plugin-import](https://www.npmjs.com/package/babel-plugin-import) - the config you highlighted is the config for that plugin, setting it to optimise our `lodash` imports so we only import the parts of the `lodash` library we actually use (so we import 3x functions, not all ~100 functions that lodash has).

Reply:
Ah I see, very nice!